stream-compositions/events/product-events/src/main/resources/events/egress/arrangement-failed-event.json

```diff
@@ -0,0 +1,10 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "properties": {
+    "message": {
+      "description": "Error message.",
```

Comment:
Is this the only thing we are getting back ? sometimes the failed ids are useful info to have in this event.